### ISSUE TITLE
Dokumenttien haku aikaleimojen perusteella

### ISSUE THREAD

**Issue by sijualle** [TIM-JYU/TIM#3498]
Tuli tilanne jossa piti etsiä mikä on vanhin dokumentti jossa käytettiin yhtä makroa. Haulla löytyi dokumentit joissa makroa käytettiin, mutta sitten piti manuaalisesti katsella milloin nuo dokumentit on luotu. Tässä olisi helpottanut jos haun tulokset olisi voinut järjestää dokumentin luomispäivän mukaan.

Toinen ehkä hyödyllinen tapa järjestää tuloksia olisi niiden viimeisen muokkaushetken mukaan.

![kuva](https://github.com/TIM-JYU/TIM/assets/63715895/6997b0d2-063b-4110-8688-894a6311e97b)

**Comment by saviit** [comment]
Joo, tuossa täytyisi lisätä nuo indeksoitavaan metadataan, sillä tällä hetkellä noita tietoja ei sisällytetä hakuindekseihin. Lisäksi tietty täytyy kirjoittaa myös sopivat sort-funktiot käyttöliittymään. Ei pitäisi olla hirveän iso homma. 

Mulla on ollut pidemmän aikaa suunnitteilla kirjoittaa tuo koko hakutoteutus uusiksi, jotta päästäisiin oikeasti järkeviin hakunopeuksiin (+ eroon indeksien jatkuvasta uudelleen rakentamisesta), tämä voisi olla järkevä sisällyttää siihen? Samalla voisi harkita tarvitseeko lisätä muitakin haku/järjestyskriteereitä.
